package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "radash",
-  "version": "9.5.0",
+  "version": "10.0.0",
```

```suggestion
  "version": "9.5.0",
```

@hovsater when you can, could you apply this change? I'm going to wrap your pr into v10 since I realized the type change could be breaking for some people on strict mode. A bit confusing but since you're going into v10 we don't want to bump the version.

---

Nevermind, it let me :)